lib/shitty_linq_ex.ex

```diff
@@ -2,7 +2,27 @@ defmodule ShittyLinqEx do
   @moduledoc """
   Documentation for `ShittyLinqEx`.
   """
+  
+  #The sum() function
+  
+  @doc """
+  Finds the sum of all values in a list with numeric elements.
+  
+  ##Examples
+  
+    iex> list = [1, 2, 3]
+    iex> shittyLinqEx.sum(list)
+    6
```

Comment:
This test looks like it would fail because of the lower-cased `shittyLinqEx` rather than using the module name `ShittyLinqEx`. Does this test run when using the `mix test` command?